lib/cli/cordova.js

```diff
@@ -36,6 +36,13 @@ module.exports = function(argv, callback) {
     
     // stop cordova-cli from tracking us!
     cordovaCommand.push("--no-telemetry");
+    if(cordovaCommand[0] == "cordova" && 
+       cordovaCommand[1] == "plugin" && 
+       cordovaCommand.indexOf("--save") == -1 && cordovaCommand.indexOf("--no-save") == -1 &&
+       cordovaCommand[2] == "add"  || cordovaCommand[2] == "remove" || cordovaCommand[2] == "rm")
+    {
```

Comment:
plugin|plugins

Maybe change 
`cordovaCommand[2] == "add"  || cordovaCommand[2] == "remove" || cordovaCommand[2] == "rm")` 
into 
`['add','remove','rm'].indexOf(cordovaCommand[2]) > -1 `

I personally prefer this to regex, this is readable ...